docs/cloud/rahti/usage/projects_and_quota.md

```diff
@@ -4,10 +4,10 @@
 
 !!! info
 
-    Projects in Rahti are separated from CSC computing projects. A single CSC
-    computing project can have access to multiple projects in Rahti.
-    Each CSC computing project with access to Rahti will have a corresponding *group* in
-    Rahti.
+  Projects in Rahti are separated from CSC computing projects. A single CSC
+  computing project can have access to multiple projects in Rahti.
+  Each CSC computing project with access to Rahti will have a corresponding *group* in
+  Rahti.
```

Comment:
A bit of a regression with the admonitions: [OpenShift projects and CSC computing projects](https://csc-guide-preview.2.rahtiapp.fi/origin/mkdocs-strict/cloud/rahti/usage/projects_and_quota/#openshift-projects-and-csc-computing-projects)

Reply:
I am not sure why I did that, but it should be fixed.